doc/tutorials/highgui/trackbar/trackbar.markdown

```diff
@@ -88,10 +88,10 @@ Explanation
 
 We only analyze the code that is related to Trackbar:
 
--#  First, we load 02 images, which are going to be blended.
+-#  First, we load two images, which are going to be blended.
     @code{.cpp}
-    src1 = imread("../../images/LinuxLogo.jpg");
-    src2 = imread("../../images/WindowsLogo.jpg");
+    src1 = imread("../data/LinuxLogo.jpg");
+    src2 = imread("../data/WindowsLogo.jpg");
     @endcode
```

Comment:
@sturkmen72 you might want to change it in other places also if it still occurs somewhere else :)